x-pack/legacy/plugins/apm/public/components/shared/TimestampTooltip/__test__/index.test.tsx

```diff
@@ -47,6 +47,16 @@ describe('asAbsoluteTime', () => {
       'Dec 1, 2019, 13:00:00.000 (UTC+1)'
     );
   });
+  it('should show end time when available', () => {
+    moment.tz.setDefault('Europe/Copenhagen');
+    expect(
+      asAbsoluteTime({
+        time: 1572266880000,
+        endTime: 1572272640000,
+        precision: 'minutes'
+      })
+    ).toBe('Oct 28, 2019, 13:48 - 15:24 (UTC+1)');
+  });
```

Comment:
What happens if you set `precision` to `days`?

```ts
      asAbsoluteTime({
        time: 1572266880000,
        endTime: 1572272640000,
        precision: 'days'
      })
```

Reply:
... and what if the interval is in years? I'm starting to think that you can't use `asAbsoluteTime` for this, since a range is inherently relative (not absolute), and the format depends on the size of the interval.

Perhaps you should make a new function `asDateRange(start, end)`

Reply:
That makes sense.